config/locales/geoblacklight.en.yml

```
@@ -29,6 +29,7 @@ en:
       data_type: 'Data type'
       placename: 'Placename'
       subject: 'Subject'
+      version: 'GeoBlacklight version'
```

Now using the version variable

```suggestion
      version: 'GeoBlacklight version %{version}'
```